pymilvus/client/constants.py

```diff
@@ -1,7 +1,7 @@
-from ..grpc_gen.common_pb2 import ConsistencyLevel
+from ..grpc_gen import common_pb2
 
 LOGICAL_BITS = 18
 LOGICAL_BITS_MASK = (1 << LOGICAL_BITS) - 1
 EVENTUALLY_TS = 1
 BOUNDED_TS = 2
-DEFAULT_CONSISTENCY_LEVEL = ConsistencyLevel.Bounded
```

Comment:
This change is due to `pylint` unable to identify the check imported from a `*.pyi` file.

Reply:
Can we define `ConsistencyLevel = common_pb2.ConsistencyLevel`, so the codes depend on the `ConsistencyLevel` can keep the compatibility?

Reply:
In this way, it seems that the changes related to the `ConsistencyLevel` can be also saved.

Reply:
It's a enum defined by proto, won't be changed from sdk, we're just using its enum functions; But I edit it anyway to reduce some code changes.